src/database/migrations/20190905181413-create-like.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Likes', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    likedBy: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
```

Comment:
Kindly explain what kind of relationship this is? 1:n or n:m? I have been having a hard time with relationships lately.

Reply:
It's a 1:1 relationship mate! 'CASCADE' on delete, deletes the item when its parent has been deleted.